portfolio/src/main/java/com/google/sps/data/entryObject.java

```diff
@@ -0,0 +1,42 @@
+package com.google.sps.data;
+
+import java.sql.Timestamp;    
+import java.util.Date;
+
+public class entryObject {
```

Comment:
Class names should always be upper camel case: https://google.github.io/styleguide/javaguide.html#s5.2.2-class-names